File: java/src/main/java/io/eventdriven/slimdownaggregates/book/entities/Chapter.java
package io.eventdriven.slimdownaggregates.book.entities;

public class Chapter {
  private final ChapterTitle title;
  private final ChapterContent content;

  public Chapter(ChapterTitle title, ChapterContent content) {
    this.title = title;
    this.content = content;
  }

  public ChapterTitle getTitle() {
    return title;
  }

  public ChapterContent getContent() {
    return content;
  }
}

File: csharp/Original/PublishingHouse.Persistence/Books/Entities/Chapter.cs
namespace PublishingHouse.Persistence.Books.Entities;

public class Chapter
{
    public Chapter(Guid id, int number, string title, string content)
    {
        Id = id;
        Number = number;
        Title = title;
        Content = content;
    }

    public Guid Id { get; }
    public int Number { get; }
    public string Title { get; }
    public string Content { get; }
}
